arcgis-ios-sdk-samples/Cloud & Portal/Integrated Windows Authentication/README.md

```diff
@@ -0,0 +1,33 @@
+# Integrated Windows Authentication
+
+Uses Windows credentials to access services hosted on a portal secured with Integrated Windows Authentication (IWA).
+
+![Screenshot of Integrated Windows Authentication sample](image1.png)
```

Comment:
I don't see this image and code in the App

Reply:
We intentionally don't show the screenshot in the readme, but the missing source code is a bug. I'll update the PR.